### ISSUE TITLE
Cannot update a model with composite keys through a referenced model

### ISSUE THREAD

**Issue by iRoachie** [prisma/prisma#4082]
## Bug description

Prisma is unable to update a model which references another model that has composite keys.

## How to reproduce

I've provided a repo https://github.com/iRoachie/bug-free-pancake

## Expected behavior

A new row in ActiveProvider table should be added to reflect the update. 

`createSite-raw.ts` does it in plain SQL as an example of what should happen.

## Prisma information

Provided in the repo

## Environment & setup

- OS: MacOs
- Database: MySQL
- Node.js version: v14.9.0
- Prisma version:

```
@prisma/cli          : 2.10.0
@prisma/client       : 2.10.0
Current platform     : darwin
Query Engine         : query-engine af1ae11a423edfb5d24092a85be11fa77c5e499c (at node_modules/@prisma/cli/query-engine-darwin)
Migration Engine     : migration-engine-cli af1ae11a423edfb5d24092a85be11fa77c5e499c (at node_modules/@prisma/cli/migration-engine-darwin)
Introspection Engine : introspection-core af1ae11a423edfb5d24092a85be11fa77c5e499c (at node_modules/@prisma/cli/introspection-engine-darwin)
Format Binary        : prisma-fmt af1ae11a423edfb5d24092a85be11fa77c5e499c (at node_modules/@prisma/cli/prisma-fmt-darwin)
Studio               : 0.304.0
```

**Comment by pantharshit00** [comment]
I can confirm this. Thanks for reporting!

**Comment by dpetrick** [comment]
Glancing over the code makes me think this is either the same or a closely related issue to https://github.com/prisma/prisma/issues/3081, will say more when I looked into it.

**Comment by dpetrick** [comment]
A fix will be available in version `2.11`.
